### PR TITLE
Repo details page

### PR DESCRIPTION
**Pull request by Aiga115**
Issue 628
Finished with repository page details 

Screenshot 2023-01-31 at 3 45 17 PM

### CODE REVIEW

**Review by Salam-Dalloul — CHANGES_REQUESTED** [review]
Thanks @Aiga115 , great job, could you please fix the pr as discussed

File: applications/osb-portal/src/assets/images/workspace-banner.png:Zone.Identifier [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[ZoneTransfer]
```
Comment (filippomc) [inline applications/osb-portal/src/assets/images/workspace-banner.png:Zone.Identifier:1]: better remove this file

Comment (filippomc) [reply applications/osb-portal/src/assets/images/workspace-banner.png:Zone.Identifier:1]: also the related workspace-banner.png doesn't seem necessary

File: applications/osb-portal/src/pages/WorkspacePageTemperoray.tsx [hunk 2]
```diff
@@ -0,0 +1,362 @@
+import * as React from "react";
```
Comment (filippomc) [inline applications/osb-portal/src/pages/WorkspacePageTemperoray.tsx:1]: name typo

**Pull request merged**